src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/HPackDecoder.cs

```diff
@@ -374,10 +394,10 @@ public void CompleteDecode()
             }
         }
 
-        private void OnIndexedHeaderField(int index, HeaderCallback onHeader, object onHeaderState)
+        private void OnIndexedHeaderField(int index, IHttpHeadersHandler handler)
         {
             HeaderField header = GetHeader(index);
-            onHeader(onHeaderState, new ReadOnlySpan<byte>(header.Name), new ReadOnlySpan<byte>(header.Value));
+            handler.OnHeader(new Span<byte>(header.Name), new Span<byte>(header.Value));
```

Comment:
Why are these changing from ReadOnlySpan to Span?  The callee is going to modify them?

Reply:
Copy pasta error. ASP.NET Core had interfaces with Span<byte>. CoreFx had ReadOnlySpan and delegate. I just did an oopsie.